src/kvstore-client/index.ts

```diff
@@ -143,7 +146,7 @@ class KVStore extends ClientBase {
    * bot.kvstore.isDeleted(res).then(isDeleted => console.log({isDeleted}))
    */
   public isDeleted(res: keybase1.KVGetResult): boolean {
-    return res.revision > 0 && res.entryValue === ''
+    return res.revision > 0 && !entryValueIsSet(res.entryValue)
```

Comment:
just to understand, do we expect entryValue==undefined if the entry is deleted?

Reply:
`undefined` is what would happen if the key were missing from the deserialized json coming out of the client (as opposed to explicitly being set to null, which is what it should be after the client PR is merged). it would be unexpected for the client to leave it out, but i'd say not irrational for the bot library to insulate itself from the client potentially doing this.